front/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+services:
+  web:
+    build:
+      context: ./
+      target: runner
+    volumes:
+      - .:/app
+    command: npm run dev
```

Comment:
maybe would be better to create docker-compose.dev.yml to run app in dev env in the future. Or just change it now to prod if running it on dev env is unnecessary